tests/conftest.py

```diff
@@ -188,4 +188,4 @@ def bento_bundle_path(bento_service):  # pylint:disable=redefined-outer-name
 
 def delete_saved_bento_service(name, version):
     yc = YataiClient()
-    yc.repository.delete(f'{name}:{version}')
+    yc.repository.delete(bento_name=name, bento_version=version, confirm_delete=True)
```

Comment:
why do we need `confirm_delete=True` in test?

Reply:
`confirm_delete=True` will skip over the click prompt.